### DISCUSSION TITLE
Love this project , want to contribute and donate  and get access to plus GH repository

### DISCUSSION THREAD

**Discussion by ksingh7** [devlikeapro/waha#108] [General]
@allburov First of all i would like to say thanks for creating this project and its very generous of you to provide almost all features in `core`. 

I would like to donate and contribute to whatsapp-http-api-plus project.
For my use case i need to create multiple sessions which is provided as a `plus` feature and i am will to make a donation to get access to `plus` , however just getting the plus docker image will not be helpful as we build our container images and sign that before deployment.

So my request is to get access to `plus`  github repository once i make the donation to support this project and say thank you.  Do you think you can add me to your ORG.

I am also willing to add new features to the plus project , if you permit 

Looking forward to your response

**Comment by allburov** [comment]
Hi, @ksingh7! Thank you for the kind words!
Yes, you can have an access to source code if you subscribe to Advanced level https://boosty.to/wa-http-api/about

Advanced
- Access to WAHA PLUS docker image.
- An invitation to a private Github repository so you have the WAHA Plus source code.
About $39 per month

Sure, we'd love to have more contributions to the project! I'll be happy to review and guide you there!
Which features are you interested in, btw?

**Comment by ksingh7** [comment]
@allburov  thanks for the response , happy to do donation of $39 and get Advance level contribution access

**Comment by ksingh7** [comment]
Some of the feature that i am interested is in
- WhatsApp Audio Calls
- WhatsApp Video Calls
- Communities

**Comment by ksingh7** [comment]
@allburov  i am about to make payment now , however it shows 39$ on one place and 45$ on another place.

I am trying this on https://boosty.to/  as agreed earlier can you pls update the 39$ monthly pricing and i will be donating right away.

<img width="728" alt="image" src="https://github.com/devlikeapro/whatsapp-http-api/assets/9701902/4ebe6b54-0eb3-4b2a-9b00-9248256e36e7">

**Reply by allburov** [reply]
Hi! 
Sorry, the actual price depends on currency conversion rate on Boosty :( 
You can get the subscription on Patreon - it's always $39 there (but it adds some taxes tho, so it'll be a bit more than $39 anyway) https://www.patreon.com/wa_http_api

**Reply by ksingh7** [reply]
@allburov  Understood, i just paid the $45 and have joined with Advance Membership, can you pls invite me to the private repository my Github ID is `ksingh7`

**Reply by ksingh7** [reply]
@allburov  still waiting for the access.

**Reply by allburov** [reply]
@ksingh7 hi! Have you accepted the invitation? I sent it yesterday  
![image](https://github.com/devlikeapro/whatsapp-http-api/assets/4376814/adb24ec0-4f7a-4610-8a7a-71c8a2686104)

**Reply by ksingh7** [reply]
@allburov  Thanks , i have accepted the invitation.

**Reply by ksingh7** [reply]
Looking forward to working with you on this project @allburov
